### DISCUSSION TITLE
NetBox DNS is compatible with NetBox 3.6 out of the box!

### DISCUSSION THREAD

**Discussion by peteeckel** [sys4/netbox-plugin-dns#43] [General]
Just a quick heads-up for early-adopters and testers: So far the migration to NetBox 3.6 does not require any changes to NetBox DNS, which is a first in the time I am working on it.

Until today, every single major (and some minor) NetBox version upgrades broke NetBox DNS in more or less catastrophic ways. This was in many cases NetBox DNS's own fault for not using the properly supported APIs and programming interfaces of NetBox, in some cases it was due to changes in the NetBox plugin interface, and in some it was just bad luck.

I'm happy that there is a new level of stability and less trouble both for the maintainer (myself) and the users of NetBox DNS. A big thank-you to the NetBox team!

**Comment by kollross** [comment]
Thank you for the hard work.

**Reply by peteeckel** [reply]
Thank you for the issues, @kollross! With your help I've been able to straighten out quite a few kinks and (hopefully) improve the plugin - qualified bug reports are exactly what is needed.

**Comment by peteeckel** [comment]
And now, after the official release of NetBox 3.6 and running the tests on it, I can confirm that NetBox DNS is also compatible with NetBox 3.6 :-)
